bitnami/schema-registry/README.md

```diff
@@ -0,0 +1,439 @@
+# Schema Registry
```

Comment:
```suggestion
<!--- app-name: WordPress -->

# Schema Registry packaged by Bitnami
```

Reply:
`WordPress` is correct or is it a typo?

Reply:
Based on other assets, I added the following:
```
<!--- app-name: Confluent Schema Registry -->

# Confluent Schema Registry packaged by Bitnami

Reply:
Yep, copy/paste error